src/main/resources/db/migration/V002__add_payment_table.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE payments_table (
+  id UUID PRIMARY KEY,
+  created_at TIMESTAMP NOT NULL,
+  envelope_id VARCHAR(50) NOT NULL,
+  ccd_reference VARCHAR(50) NOT NULL,
+  jurisdiction VARCHAR(50) NOT NULL,
+  service VARCHAR(50) NOT NULL,
+  po_box VARCHAR(50),
```

Comment:
Out of interest, is the PO_BOX optional? If so thats fine

Reply:
updated PO_BOX is required